### PR TITLE
MNT: Adding missing setuptools.

### PR DESCRIPTION
**Pull request by zssherman**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mgrover1** [comment]
@zssherman looks like we need to add cython + versioner when using `pip check`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,6 +40,10 @@ requirements:
 test:
   imports:
     - pyart
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (mgrover1) [inline recipe/meta.yaml:49]: We need to add `cython` and `versioneer` here

**Review by mgrover1 — CHANGES_REQUESTED** [review]

**Comment by zssherman** [comment]
@mgrover1 ah good catch, I'll do that change shortly

**Review by mgrover1 — APPROVED** [review]
Looks like it built everything correctly and passed all the tests!

**Comment by beckermr** [comment]
You need to be using pip on windows too. 

https://github.com/conda-forge/arm_pyart-feedstock/blob/main/recipe/bld.bat

**Comment by zssherman** [comment]
@beckermr the bld.bat should be instead?:
"%PYTHON%" -m pip install . -vvv

**Comment by beckermr** [comment]
yeah or even better just remove the not win selector in the meta.yaml.

**Comment by zssherman** [comment]
Gotcha sounds good @beckermr, I will remove the not win selector in the meta.yml is it better to do a new build PR and then mark both PRs windows builds as broken?

**Comment by beckermr** [comment]
Let's ensure the new one works and inspect the artifacts. Then we can merge it and mark the old ones as broken.

**Comment by zssherman** [comment]
Sounds good!

**Pull request merged**